src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -72,9 +80,14 @@ public function __construct(
         $this->toStringCallback = $toStringCallback;
     }
 
+    /**
```

Comment:
We could add a `@param`.

There is a check
```
if (!\is_array($value)) {
     throw new \UnexpectedValueException(sprintf('Value should be array, %s given.', \gettype($value)));
}
```

So `@param array` seems ok.

There is a check for empty values before the array check, 
but the symfony doc say
```
     * This method must be able to deal with empty values. Usually this will
     * be an empty string, but depending on your implementation other empty
     * values are possible as well (such as NULL). The reasoning behind
     * this is that value transformers must be chainable. If the
     * reverseTransform() method of the first value transformer outputs an
     * empty string, the second value transformer must be able to process that
     * value.
```

So it would means that allowed values are arrays, `null` and `''` ?